<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlpaintClass</class>
 <widget class="QMainWindow" name="AlpaintClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alpaint</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#AlpaintClass
{
	background-color: rgb(40, 40, 40);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget
{
	background-color: rgb(40, 40, 40);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowPixelGrid"/>
    <addaction name="actionShowBackground"/>
   </widget>
   <widget class="QMenu" name="menuDocument">
    <property name="title">
     <string>Document</string>
    </property>
    <addaction name="actionResizeCanvas"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDocument"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="alp::ToolBarDock" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>68</width>
     <height>524287</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QDockWidget#toolBar
{
	border-top: 1px solid;
	border-right: 1px solid;
	background-color: rgb(83, 83, 83);
}</string>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="toolBarContents">
    <property name="styleSheet">
     <string notr="true">QWidget#toolBarContents
{
	background-color: rgb(83, 83, 83);
	border: 1px solid rgb(56, 56, 56);
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="colorTool">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="alp::ColorPicker" name="primaryColor">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>35</width>
          <height>35</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:black;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="alp::ColorPicker" name="secondaryColor">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>15</y>
          <width>35</width>
          <height>35</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <zorder>secondaryColor</zorder>
       <zorder>primaryColor</zorder>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="pencilButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
	background-color: rgb(50, 50, 50);
	border: 1px solid rgb(45, 45, 45);
	border-radius: 3px;
}
QToolButton:hover
{
	background-color: rgb(70, 70, 70);
};</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/pencil.png</normaloff>icons/pencil.png</iconset>
         </property>
         <property name="shortcut">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="eraserButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
	background-color: rgb(50, 50, 50);
	border: 1px solid rgb(45, 45, 45);
	border-radius: 3px;
	color: white;
}
QToolButton:hover
{
	background-color: rgb(70, 70, 70);
};</string>
         </property>
         <property name="text">
          <string>E</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/eraser.png</normaloff>icons/eraser.png</iconset>
         </property>
         <property name="shortcut">
          <string>E</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="rectButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
	background-color: rgb(50, 50, 50);
	border: 1px solid rgb(45, 45, 45);
	border-radius: 3px;
	color: white;
}
QToolButton:hover
{
	background-color: rgb(70, 70, 70);
};</string>
         </property>
         <property name="text">
          <string>R</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/rect.png</normaloff>icons/rect.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ellipseButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
	background-color: rgb(50, 50, 50);
	border: 1px solid rgb(45, 45, 45);
	border-radius: 3px;
	color: white;
}
QToolButton:hover
{
	background-color: rgb(70, 70, 70);
};</string>
         </property>
         <property name="text">
          <string>O</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/ellipse.png</normaloff>icons/ellipse.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="colorPickerButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
	background-color: rgb(50, 50, 50);
	border: 1px solid rgb(45, 45, 45);
	border-radius: 3px;
	color: white;
}
QToolButton:hover
{
	background-color: rgb(70, 70, 70);
};</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/pipette.png</normaloff>icons/pipette.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="fillButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
	background-color: rgb(50, 50, 50);
	border: 1px solid rgb(45, 45, 45);
	border-radius: 3px;
	color: white;
}
QToolButton:hover
{
	background-color: rgb(70, 70, 70);
};</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/fill.png</normaloff>icons/fill.png</iconset>
         </property>
         <property name="shortcut">
          <string>F</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="alp::InformationBarDock" name="informationBar">
   <property name="styleSheet">
    <string notr="true">QDockWidget#informationBar
{
	border-top: 1px solid;
	border-right: 1px solid;
	background-color: rgb(85, 85, 85);
}</string>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="informationBarContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#informationBarContents
{
	background-color: rgb(83, 83, 83);
}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionShowPixelGrid">
   <property name="text">
    <string>Show pixel grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionShowBackground">
   <property name="text">
    <string>Show background</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionResizeCanvas">
   <property name="text">
    <string>Resize canvas...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>alp::ToolBarDock</class>
   <extends>QDockWidget</extends>
   <header>docks/ToolBarDock.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>alp::ColorPicker</class>
   <extends>QFrame</extends>
   <header>toolbar/ColorPicker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>alp::InformationBarDock</class>
   <extends>QDockWidget</extends>
   <header>docks/InformationBarDock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Alpaint.qrc"/>
 </resources>
 <connections/>
</ui>
